File: sample/src/main/java/com/schibsted/spain/barista/sample/SpinnerActivity.java
package com.schibsted.spain.barista.sample;

import android.os.Bundle;
import androidx.appcompat.app.AppCompatActivity;
import android.widget.ArrayAdapter;
import android.widget.Spinner;

public class SpinnerActivity extends AppCompatActivity {

  private static final String[] FRUITS = { "Banana", "Apple", "Orange", "Raspberry" };

  @Override
  protected void onCreate(Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_spinner);

    Spinner spinner = (Spinner) findViewById(R.id.spinner);
    spinner.setAdapter(new ArrayAdapter(this, android.R.layout.simple_list_item_1, FRUITS));
  }
}

File: sample/src/main/java/com/schibsted/spain/barista/sample/SpinnerActivity.kt
package com.schibsted.spain.barista.sample

import android.os.Bundle
import android.view.View
import android.widget.ArrayAdapter
import android.widget.Spinner
import androidx.appcompat.app.AppCompatActivity

class SpinnerActivity : AppCompatActivity() {
  override fun onCreate(savedInstanceState: Bundle?) {
    super.onCreate(savedInstanceState)
    setContentView(R.layout.activity_spinner)
    val spinner = findViewById<View>(R.id.spinner) as Spinner
    spinner.adapter = ArrayAdapter<Any?>(this, android.R.layout.simple_list_item_1, FRUITS)
  }

  companion object {
    private val FRUITS = arrayOf<String?>("Banana", "Apple", "Orange", "Raspberry")
  }
}